guides/welcome.md

```diff
@@ -39,4 +41,4 @@ If you want to run the following code yourself, there are three options:
 >
 > You can navigate through the pages with the `left` and `right` arrow
 
-Let's begin: [Learn how to write 'Hello World'](guides/01-basics/hello-world.livemd)
\ No newline at end of file
+Let's begin: [Learn how to write 'Hello World'](guides/01-basics/hello-world.livemd)
```

Comment:
I used the builtin Github editor to add the 2 lines above and I have no idea why this unchanged line is in the commit.